generator/internal/discovery_to_proto.cc

```diff
@@ -218,6 +218,46 @@ StatusOr<DiscoveryTypeVertex> SynthesizeRequestType(
   return DiscoveryTypeVertex(id, synthesized_request);
 }
 
+Status ProcessMethodRequestsAndResponses(
+    std::map<std::string, DiscoveryResource>& resources,
+    std::map<std::string, DiscoveryTypeVertex>& types) {
+  for (auto& resource : resources) {
+    std::string resource_name = resource.first;
```

Comment:
**optional:** Should we just capitalize the resource name here? (Instead of doing it for each `method`). The only difference is the error info.